### ISSUE TITLE
QtWebengine: regression with qtwebengine_bin-5.15.16-4-x86_64.hpkg

### ISSUE THREAD

**Issue by korli** [haikuports/haikuports#9983]
With qtwebengine_bin-5.15.2-8-x86_64.hpkg the video stream https://www.france.tv/franceinfo/direct.html works successfully, with  qtwebengine_bin-5.15.16-4-x86_64.hpkg, the stream doesn't start.

**Comment by threedeyes** [comment]
I can't reproduce this bug. This stream works fine for me with 5.15.16-4 qtwebengine version.

https://github.com/haikuports/haikuports/assets/2559260/c9bd6dee-fc3a-4cce-96bb-2d7e1652abb7

**Comment by korli** [comment]
It works partly then stop with:
[8336:2110737120:0113/185419.310058:ERROR:batching_media_log.cc(38)] MediaEvent: {"error":"Append: stream parsing failed. Data size=95819 append_window_start=0 append_window_end=inf"}
[8336:2110737120:0113/185419.320974:ERROR:batching_media_log.cc(35)] MediaEvent: {"pipeline_error":16}

Are you on latest nightly?

**Comment by korli** [comment]
BTW it seems the QtWebEngine version is hardcoded in Falkon, switching to an older package doesn't change the shown version About popup.

**Comment by threedeyes** [comment]
> BTW it seems the QtWebEngine version is hardcoded in Falkon, switching to an older package doesn't change the shown version About popup.

Yeah, that's right.  The version string is generated during compilation - 
https://github.com/KDE/falkon/blob/master/src/lib/other/aboutdialog.cpp#L58C77-L58C104

**Comment by threedeyes** [comment]
> Are you on latest nightly?

Yes

**Comment by threedeyes** [comment]
The error is reproduced on different sites with online streaming. Time to error depends on the quality of the video stream. The bug is not reproduced on low quality.
